Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -606,7 +604,22 @@ namespace Multiplayer
         [[maybe_unused]] MultiplayerPackets::ClientMigration& packet
     )
     {
-        return false;
+        if (GetAgentType() != MultiplayerAgentType::Client)
+        {
+            // Only clients are allowed to migrate from one server to another
+            return false;
+        }
+
+        // Store the temporary user identifier so we can transmit it with our next Connect packet
+        // The new server will use this to re-attach our set of autonomous entities
```

Comment:
This doesn't seem to be happening?

Reply:
Yup, currently refactoring this whole thing since I want client migration to be robust in the event of a server crash.